xlators/mgmt/glusterd/src/glusterd.h

```diff
@@ -802,6 +802,64 @@ typedef ssize_t (*gd_serialize_t)(struct iovec outmsg, void *args);
                                                                                \
     } while (0)
 
+#define CLAUSE_LABEL "CLAUSE"
+
+struct opval_dep;
+typedef struct opval_dep opval_dep_t;
+
+struct opval_dep {
+    char *op;
+    char *val;
```

Comment:
The type of `val` should be an enum, which ranges over 0, 1 (indicating Booleans) and 2 (indicating clause). (Thus also eliminating the `CLAUSE_LABEL` macro.)